package.json

```diff
@@ -111,7 +111,7 @@
     "react-native-listener": "1.0.1",
     "react-phone-number-input": "^0.15.2",
     "react-redux": "4.4.6",
-    "react-webcam-onfido": "^0.1.10",
+    "react-webcam-onfido": "git://github.com/onfido/react-webcam.git#feature/fix-ios-safari",
```

Comment:
Looks like the `flow` and npm config in this repo are set up in such a way that this doesn't work :/

So you'll have to build this locally to preview the change...

But yes, all the actual _code_ to review is [in this PR](https://github.com/onfido/react-webcam/pull/29)

Reply:
Feel free to publish `react-webcam-onfido` when it's approved and the acceptance can be done via this PR